buildlib/pr/codestyle.yml

```diff
@@ -70,3 +70,25 @@ jobs:
               azure_complete_with_issues "Code is not formatted according to the code style, see $url for more info."
           fi
         condition: eq(variables['Build.Reason'], 'PullRequest')
+
+  - job: codespell
+    displayName: codespell check
+    pool:
+      name: MLNX
+      demands:
+      - ucx_docker -equals yes
+    container: fedora
+    steps:
+      - checkout: self
+        clean: true
+        fetchDepth: 100
+        retryCountOnTaskFailure: 5
```

Comment:
it's a minor comment, but maybe we do not need to set this values to be that large. @Alexey-Rivkin what are the defaults for `fetch` and `retry`? Can we just use defaults here?

Reply:
The default for retry is 0, we made it 5 to overcome pure connectivity issues.
A fetch depth of 100 is Ok too, less might fail with the "commit not found" error if the PR was shelved for a while.